# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../mythtv/mythtv-0.24-libmythhdhomerun-use-ldflags.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix to use extra LDFLAGS while linking libmythhdhomerun

MythTV provides a configure option '--extra-ldflags' to pass extra LDFLAGS
which is handy if a library is not installed in a path supported by default.
On OpenSDE based systems X11R7 gets installed with the prefix '/usr/X11R7'
so libraries are installed in '/usr/X11R7/lib', which is not found by the
MythTV build system, thus we are passing '-L/usr/X11R7/lib' using the
'--extra-ldflags' configure option.
This seems to work fairly well, besides with libmythhdhomerun which is not
honoring the provided extra LDFLAGS.
 
--- mythtv-0.24/libs/libmythhdhomerun/libmythhdhomerun.pro.orig	2011-04-08 12:48:54.436075914 +0200
+++ mythtv-0.24/libs/libmythhdhomerun/libmythhdhomerun.pro	2011-04-08 12:14:57.184075914 +0200
@@ -33,6 +33,6 @@
     LIBS += -lws2_32 -liphlpapi -lpthread
 }
 
-LIBS += $$EXTRALIBS 
+LIBS += $$LDFLAGS $$EXTRALIBS 
 
 include ( ../libs-targetfix.pro )
